src/saving/SaveManagerGUI.cs

```diff
@@ -127,6 +127,13 @@ public override void _Process(float delta)
         refreshing = false;
     }
 
+    public void OnBackButton()
```

Comment:
Based on my previous comments, changing this to public is unnecessary.

Reply:
The function I wrote myself needs to call this. The two of them are not in the same class, so they can only be set to public.

Reply:
See my previous commit. I don't think this is a good design, so this method can be left private.